### DISCUSSION TITLE
using it

### DISCUSSION THREAD

**Discussion by ikke1234** [SebLague/Digital-Logic-Sim#43] [General]
how do i add the code to the program

**Comment by Subroutine7901** [comment]
what do you mean by code? if you want to run code _in_ the program then you can't, if you want to _improve_ on the program then follow this tutorial:
[link to tutorial](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
hope this helps

**Reply by Subroutine7901** [reply]
and you'd probably have to open the code in unity and edit it there
